.vscode/settings.json

```diff
@@ -53,5 +53,9 @@
       "fileMatch": ["test/visual-diff/visual-tests.jsonc"],
       "url": "./build-system/json-schemas/visual-tests.json"
     }
-  ]
+  ],
+  "editor.bracketPairColorization.enabled": true,
```

Comment:
@danielrozenberg could you take a look at these vscode settings change

Reply:
I think I would prefer if we don't make this change. I personally have the same setting, but in my user profile - we should only have settings that are project-specific in this file :)

Reply:
@danielrozenberg agree. @mmprebid could you revert the changes in this file. thanks

Reply:
@erwinmombay, 
Thanks for the replying
As suggested, we have revert the .vscode settings to previous. Please take a look and let us know for any update.